src/App.js

```diff
@@ -1,12 +1,24 @@
 import React, { Component } from 'react';
+import './app.scss';
+//import { Button } from 'carbon-components-react';
```

Comment:
It is best practice to delete unneeded code rather than comment it out.

Reply:
Ok sure I'll fix that. I got an warning during compilation that Button was being unused and was unsure if we were going to use it again, so I had commented it out temporarily. Thanks